bots/bridgekeeper/src/main/java/org/openjdk/skara/bots/bridgekeeper/PullRequestCloserBot.java

```diff
@@ -52,13 +54,24 @@ private void checkWelcomeMessage() {
                                     .anyMatch(comment -> comment.body().contains(welcomeMarker));
 
         if (!welcomePosted) {
-            var message = "Welcome to the OpenJDK organization on GitHub!\n\n" +
-                    "This repository is currently a read-only git mirror of the official Mercurial " +
-                    "repository (located at https://hg.openjdk.java.net/). As such, we are not " +
-                    "currently accepting pull requests here. If you would like to contribute to " +
-                    "the OpenJDK project, please see https://openjdk.java.net/contribute/ on how " +
-                    "to proceed.\n\n" +
-                    "This pull request will be automatically closed.";
+            String message = null;
+            if (type == PullRequestCloserBot.Type.MIRROR) {
+                message = "Welcome to the OpenJDK organization on GitHub!\n\n" +
+                "This repository is currently a read-only git mirror of the official Mercurial " +
+                "repository (located at https://hg.openjdk.java.net/). As such, we are not " +
+                "currently accepting pull requests here. If you would like to contribute to " +
+                "the OpenJDK project, please see https://openjdk.java.net/contribute/ on how " +
+                "to proceed.\n\n" +
+                "This pull request will be automatically closed.";
+            } else if (type == PullRequestCloserBot.Type.DATA) {
+                message = "Welcome to the OpenJDK organization on GitHub!\n\n" +
+                "This repository currently holds only automatically generated data and therefore does not accept pull requests." +
+                "This pull request will be automatically closed.";
+            } else {
+                message = "Welcome to the OpenJDK organization on GitHub!\n\n" +
```

Comment:
Guess this can't be reached?

Reply:
No, not at the moment, but I figured I would provide a fallback :smile: